src/organisms/Accordion/AccordionPanel/__tests__/accordion_panel.spec.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import { mount } from 'enzyme';
+
+// import sinon from 'sinon';
```

Comment:
why was this added?

Reply:
Fixed this - accidentally added it back in when I rebased.